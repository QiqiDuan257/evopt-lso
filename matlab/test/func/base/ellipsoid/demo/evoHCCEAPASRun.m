clear; clc;

% nohup matlab -nodesktop -nodisplay -nosplash <evoHCCEAPASRun.m >evoHCCEAPASRun.out 2>&1 &

startRun = tic;
rng(20190101); % for reproducibility
funcParam.funcName = 'evoFuncEllipsoid';
% funcParam.funcDims = [2 10];
algoParam.algoName = 'evoHCCEAPAS';
[statOptRes, funcParams, algoParams] = ...
    evoCreateFAEBaseFunc(funcParam, algoParam);
fprintf(sprintf('$-runtime-$: %7.5e.\n', toc(startRun)));
