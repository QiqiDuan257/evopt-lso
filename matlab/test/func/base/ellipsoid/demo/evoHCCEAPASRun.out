*** funcDim: 2 + algoName: evoHCCEAPAS + maxFEval: 200000 ***
    opty = +9.1081e-11 | runtime = 3.29e-01 | timeFEval = 1.18e-01 | numFEval = 789 | timeCcFEval = 2.91e-03 | stopFlag = stopy
*** funcDim: 10 + algoName: evoHCCEAPAS + maxFEval: 1000000 ***
    opty = +7.3794e-11 | runtime = 4.86e-01 | timeFEval = 1.07e-01 | numFEval = 11213 | timeCcFEval = 1.56e-02 | stopFlag = stopy
*** funcDim: 100 + algoName: evoHCCEAPAS + maxFEval: 10000000 ***
    opty = +9.8496e-11 | runtime = 4.02e+00 | timeFEval = 1.40e+00 | numFEval = 182921 | timeCcFEval = 1.29e-01 | stopFlag = stopy
*** funcDim: 1000 + algoName: evoHCCEAPAS + maxFEval: 100000000 ***
    opty = +8.5589e-11 | runtime = 3.61e+02 | timeFEval = 1.82e+02 | numFEval = 9588959 | timeCcFEval = 9.04e+00 | stopFlag = stopy
*** funcDim: 2000 + algoName: evoHCCEAPAS + maxFEval: 200000000 ***
    opty = +9.9582e-11 | runtime = 1.50e+03 | timeFEval = 8.47e+02 | numFEval = 32413291 | timeCcFEval = 3.48e+01 | stopFlag = stopy
*** funcDim: 4000 + algoName: evoHCCEAPAS + maxFEval: 400000000 ***
    opty = +9.8671e-11 | runtime = 5.12e+03 | timeFEval = 3.17e+03 | numFEval = 112133542 | timeCcFEval = 9.63e+01 | stopFlag = stopy
*** funcDim: 6000 + algoName: evoHCCEAPAS + maxFEval: 600000000 ***
    opty = +1.8452e-10 | runtime = 7.20e+03 | timeFEval = 4.80e+03 | numFEval = 125832318 | timeCcFEval = 1.24e+02 | stopFlag = stopt
*** funcDim: 8000 + algoName: evoHCCEAPAS + maxFEval: 800000000 ***
    opty = +4.4111e-10 | runtime = 7.20e+03 | timeFEval = 4.94e+03 | numFEval = 123455424 | timeCcFEval = 1.11e+02 | stopFlag = stopt
*** funcDim: 10000 + algoName: evoHCCEAPAS + maxFEval: 1000000000 ***
    opty = +1.5739e-09 | runtime = 7.20e+03 | timeFEval = 5.43e+03 | numFEval = 81506348 | timeCcFEval = 1.04e+02 | stopFlag = stopt
$-runtime-$: 2.86661e+04.
